Block/Product/YotpoReviewRenderer.php

```diff
@@ -18,12 +18,26 @@ public function getReviewsSummaryHtml(
         $displayIfNoReviews = false
     ) {
         
+        $objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+        $storeManager = $objectManager->get('\Magento\Store\Model\StoreManagerInterface');
+        $cp = $objectManager->get('\Magento\Framework\App\Request\Http');
         $enableBottomlineCategoryPage = $this->isBottomlineCategoryEnabled();
-	$enableMagentoDefaultReviews = $this->isMagentoDefaultReviewsEnabled();
+		$enableMagentoDefaultReviews = $this->isMagentoDefaultReviewsEnabled();
+		
         if ($enableBottomlineCategoryPage) {
-            return $this->showCategoryBottomLine($product);
+			if ($cp->getFullActionName()==='cms_index_index'){
+				return $this->showCategoryBottomLine($product);
+			}
+			if ($cp->getFullActionName()==='catalog_category_view'){
+				return $this->showCategoryBottomLine($product);
+			}
         } elseif (!$enableMagentoDefaultReviews) {
-            return parent::getReviewsSummaryHtml($product, $templateType, $displayIfNoReviews);
+            if ($cp->getFullActionName()==='cms_index_index'){
```

Comment:
the same comment as above consider using the variable $actionName = $cp->getFullActionName()